src/collection-browser.ts

```diff
@@ -43,6 +43,7 @@ import {
   sortOptionFromAPIString,
   SORT_OPTIONS,
   defaultProfileElementSorts,
+  FacetLoadStrategy,
```

Comment:
@latonv here we are following different case styling for all the models. Somewhere we using all caps or PascalCase or camelCase. can we please use one standard for these? we can just rename other as well in this PR?

Reply:
I don't think it would make sense for these all to use the same casing, because they represent different things that are _meant_ to be cased differently in our conventions. For instance, `FacetLoadStrategy` is a type, while `sortOptionFromAPIString` is a function, and `SORT_OPTIONS` is an exported module-level constant. These tend to be distinguished visually by giving them different casing so that it is clearer at a glance what the identifier represents. Some further reading: https://google.github.io/styleguide/tsguide.html#naming-rules-by-identifier-type

One thing that might help, though, is re-organizing some of these exports into separate, smaller modules so that we don't have all of the above coming from the same place. (That project probably warrants a separate PR)